### ISSUE TITLE
Tutorial not working with ver 3.5

### ISSUE THREAD

**Issue by ashutoshsharmaindia** [streamsets/tutorials#96]
I am trying to work with the basic tutorial involving Kafka:
https://github.com/streamsets/tutorials/blob/master/tutorial-2/directory_to_kafkaproducer.md

It's not working. It seems that you user interface(UI) has changed or evolved but the tutorial is still referring to the old user interface and steps. 

The moment I am at this step:
Defining the Kafka Producer

There is no way I can follow it as the user interface and the navigation is not in sync with this documentation.
Products like stream sets which are having sucha heavy duty user interface and the navigation should be having almost the perfect documentation and updated as per the latest versions. Or may be the tutorial should clearly specify with what versions it is going to work?

**Comment by iamontheinet** [comment]
Hi @ashutoshsharmaindia,

Thanks for pointing it out. We're doing our best to keep the tutorials up-to-date and will add this one to the list for sure. In the meantime... I did a quick comparison between the UI presented in this tutorial and the latest released version (3.5) and the only discrepancy I saw under "**Defining the Kafka Producer**" was "Data Format" which has been moved to its own tab. See below:

<img width="1011" alt="screen shot 2018-10-05 at 2 16 31 pm" src="https://user-images.githubusercontent.com/1723932/46560669-72737e80-c8a9-11e8-9a4f-97b0c7d0b92b.png">

Other than that, you should be able to follow along. If not, please let me know.

Cheers,
Dash

**Comment by wiwengweng** [comment]
yes, a total of waste time, because of the out-dated documents. I wish the docs are able to be updated, so that new version of SDC can work.

**Comment by metadaddy** [comment]
@wiwengweng Where do you see the discrepancy? @iamontheinet updated the documentation over a year ago. I just checked the tutorial with the current version and it looks ok to me.

**Comment by metadaddy** [comment]
@ashutoshsharmaindia I see no screenshot in your second comment
